src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,7 +1,22 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class UserService {
+
     public void registerUser(User user) {
+        PasswordValidator passwordValidator = new PasswordValidator();
+        boolean passwordIsCorrect;
+        try {
+            passwordIsCorrect = passwordValidator.validate(user.getPassword(),
+                    user.getRepeatPassword());
+        } catch (PasswordValidationException e) {
```

Comment:
remember that if in the middle of your `try` block an exception is thrown, the rest of the `try` block would not be executed